hazelcast/src/main/java/com/hazelcast/client/impl/protocol/parameters/ExceptionResultParameters.java

```diff
@@ -33,24 +34,23 @@
      * ClientMessageType of this message
      */
     public static final int TYPE = ResponseMessageConst.EXCEPTION;
-    public String className;
-    public String causeClassName;
+    public int errorCode;
```

Comment:
Do we need a field such as human readable error name? (Would be nice for documentation)